src/modules/src/pid.c

```diff
@@ -66,15 +66,17 @@ float pidUpdate(PidObject* pid, const float measured, const bool updateError)
     output += pid->outP;
 
     float deriv = (pid->error - pid->prevError) / pid->dt;
-    if (pid->enableDFilter)
+    /*if (pid->enableDFilter)
```

Comment:
Should this code be removed. Commented code is always hard to understand.

Reply:
has been uncommented and added back in with the addition of ifdefs